l10n_nl_base_ubl/tests/test_ubl_post_install.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+import odoo
+from odoo.tests.common import Form, HttpCase
+
+
+@odoo.tests.tagged("post_install", "-at_install")
+class TestUblInvoice(HttpCase):
+    def setUp(self):
```

Comment:
Same setup as far as I can see in the other class. Maybe refactor todo the setUp in a common parent class?